<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 823, in lint_file
    recipe = _recipe.Recipe.from_file(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 243, in from_file
    raise exc from exc
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 234, in from_file
    recipe._load_from_string(text.read())
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 170, in _load_from_string
    self.render()
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 290, in render
    self.meta = renderer_utils.render(self.recipe_dir, self.dump(), self.selector_dict, self.renderer)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/_renderer.py", line 234, in render
    raise JinjaRenderFailure(recipe_dir, message=exc.message, line=exc.lineno) from exc
percy.render.exceptions.JinjaRenderFailure: (PosixPath('/tmp/abs_a70vy7jl8x/clone/recipe'), 'truncated \\uXXXX escape (at line 67)')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 840, in lint_file
    recipe = _recipe.Recipe(recipe_name)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 107, in __init__
    self.recipe_dir = recipe_file.parent
                      ^^^^^^^^^^^^^^^^^^
AttributeError: 'str' object has no attribute 'parent'

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_a70vy7jl8x/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -64,7 +64,7 @@ test:
     {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/stores/test_in_memory.py" %}
     {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/vectorstores/test_in_memory.py" %}
     {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/indexing/test_indexing.py" %} # Failing due to upstream bug https://github.com/langchain-ai/langchain/issues/22135
-    {% set pytest_args = pytest_args + ' -k "not test_as_import_path"' %}
+    {% set pytest_args = pytest_args + ' -k "not (test_as_import_path or test_benchmark_model)"' %} # Our env paths don't match expected
     - pytest -vv tests/unit_tests {{ pytest_args }}  # [not (osx and x86 and py>=312)]
 
 about:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -63,8 +63,7 @@ test:
     {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/prompts/test_prompt.py" %}
     {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/stores/test_in_memory.py" %}
     {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/vectorstores/test_in_memory.py" %}
-    # Tests below are failing due to upstream bug https://github.com/langchain-ai/langchain/issues/22135
-    {% set pytest_args = pytest_args + " --ignore=tests\unit_tests\indexing\test_indexing.py" %}
+    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/indexing/test_indexing.py" %} # Failing due to upstream bug https://github.com/langchain-ai/langchain/issues/22135
     {% set pytest_args = pytest_args + ' -k "not test_as_import_path"' %}
     - pytest -vv tests/unit_tests {{ pytest_args }}  # [not (osx and x86 and py>=312)]
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -58,14 +58,13 @@ test:
     {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/runnables/test_fallbacks.py" %}
     {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/runnables/test_graph.py" %}
     {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/runnables/test_runnable.py" %}
-    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/indexing/test_in_memory_indexer.py" %}
-    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/prompts/test_chat.py" %}
-    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/prompts/test_prompt.py" %}
-    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/stores/test_in_memory.py" %}
-    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/vectorstores/test_in_memory.py" %}
-    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/indexing/test_indexing.py" %} # Failing due to upstream bug https://github.com/langchain-ai/langchain/issues/22135
-    {% set pytest_args = pytest_args + ' -k "not (test_as_import_path or test_benchmark_model)"' %} # Our env paths don't match expected
-    - pytest -vv tests/unit_tests {{ pytest_args }}  # [not (osx and x86 and py>=312)]
+    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/indexing/test_in_memory_indexer.py" %} # missing langchain_standard_tests
+    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/prompts/test_chat.py" %} # missing syrupy
+    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/prompts/test_prompt.py" %} # missing syrupy
+    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/stores/test_in_memory.py" %} # missing langchain_standard_tests
+    {% set pytest_args = pytest_args + " --ignore=tests/unit_tests/vectorstores/test_in_memory.py" %} # missing langchain_standard_tests
+    {% set pytest_args = pytest_args + ' -k "not test_as_import_path"' %} # tarball paths don't match expected
+    - pytest -vv tests/unit_tests {{ pytest_args }}  # [(not win) or (not (osx and x86 and py>=312))]
 
 about:
   home: https://github.com/langchain-ai/langchain

